matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/timeline/SendingEventsDataSource.kt

```diff
@@ -72,7 +72,8 @@ internal class RealmSendingEventsDataSource(
         if (frozenSendingTimelineEvents?.isValid == true) {
             frozenSendingTimelineEvents?.realm?.close()
         }
-        frozenSendingTimelineEvents = sendingEvents?.freeze()
+        // Do not freeze empty list
+        frozenSendingTimelineEvents = sendingEvents?.takeIf { it.isNotEmpty() }?.freeze()
```

Comment:
maybe assign emptyList?

Reply:
it does not change anything and probably a tiny bit less efficient.